tests/async/test_async_fhir_client_patient_list.py

```diff
@@ -31,9 +31,14 @@ async def test_fhir_client_patient_list_async() -> None:
         response=mock_response(body=response_text),
         timing=times(1),
     )
+    additional_request_headers = {
+        "TestHeaderOne": "abcdtest",
+        "User-Agent": "TestPipelineName",
+    }
 
     fhir_client = FhirClient()
     fhir_client = fhir_client.url(absolute_url).resource("Patient")
+    fhir_client = fhir_client.additional_request_headers(additional_request_headers)
```

Comment:
1.Here you could have added headers in log when the loginfo level is set to debug so that we can debug in future easily by using a script if we get any bug and verify in testcases as well. here it is not getting tested properly that if we are sending additional header in our http request or not
2. Also you can do testing of this feature and be sure 100% by using this script https://icanbwell.atlassian.net/browse/HDE-2456